e2e/components/PageHeader/PageHeader-test.avt.e2e.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
```

Comment:
```suggestion
 * Copyright IBM Corp. 2016, 2025
```

Reply:
Should it be the following instead?

```suggestion
 * Copyright IBM Corp. 2025
```

Internally, the general rules are:

1. The first year in the notice is the year that the file was created because you can't apply a license to a file that didn't exist before it was created
2. The second year is the year that the file was last updated regardless of what that update was (i.e., add code, delete code, update code, etc.). If the second year is the same as the first, it should be omitted.

I have some changes in my fork (e.g., https://github.com/carbon-design-system/carbon/compare/2601d361089b0afd734f87defa0c5039ec8b943c...e555565119e169f20e6bc7d1de1afd02c76aece1 which I was working on earlier in the week) where I'm revisting all of these copyright notices because the vast majority of them are wrong. I don't know what kind of legal implications there are for these issues so I figure it's better to get ahead of it and fix as much as possible now.